packages/blocks/code/src/webpack-dev-server.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * This is the entry point for developing and debugging.
+ * This file is not bundled with the library during the build process.
+ */
+import React from "react";
+import ReactDOM from "react-dom";
+import { tw } from "twind";
+import { MockBlockDock } from "mock-block-dock";
+
+// eslint-disable-next-line import/extensions
```

Comment:
This is no longer necessary

Reply:
Done: https://github.com/hashintel/hash/pull/415/commits/2409f56abeac464729193192ad076d44493ed83d